snmp/datadog_checks/snmp/data/default_profiles/_hp-base.yaml

```diff
@@ -1,8 +1,6 @@
 metadata:
   device:
     fields:
-      vendor:
-        value: "hp"
       serial_number:
```

Comment:
I think having both _hp-base.yaml and _hp.yaml might create some confusion.

Maybe we can discuss more about best way to handle cases like this one.